source/guides/network/running-a-test-net-node.rst

```diff
@@ -9,154 +9,196 @@
 
 .. code-block:: bash
 
-    git clone https://github.com/nemgrouplimited/symbol-testnet-bootstrap.git
+    symbol-bootstrap start -p testnet -a dual
 
-2. Choose the **assembly distribution** you want to install.
+****************
+Running the node
+****************
 
-In short, if you want to be able to interact with your node, you need to run the :ref:`API assembly <api-node>`.
-On the other hand, if you want a node dedicated exclusively confirm transactions, deploy the :ref:`Peer assembly <peer-node>`.
+You really don't need to use anything else but ``symbol-bootstrap start``.  Use any of the commands above to instantiate and boot a |codename| node and ``Ctrl+C`` to shut it down.
 
-.. code-block:: bash
+Alternatively, you can start in detached mode (``--detached``) to run in the background.
 
-    cd symbol-testnet-bootstrap/api-harvest-assembly
+***************************
+Retrieving the node account
+***************************
 
-or...
+The node you just created has an associated |codename| account which you can use to interact with the node. For instance, you need this account to provide funds to the node so it can emit transactions.
 
-.. code-block:: bash
+The account's keys and address can be retrieved from a YAML file in the ``config`` folder:
 
-    cd symbol-testnet-bootstrap/peer-assembly
+``target/config/generated-addresses/addresses.yml``
 
-3. Run the node with **docker-compose**.
+As an example:
 
-.. code-block:: bash
+.. code-block:: yaml
+
+    networkType: 152
+    nemesisGenerationHashSeed: 6C1B92391CCB41C96478471C2634C111D9E989DECD66130C0430B5B8D20117CD
+    nodes:
+        -
+            type: peer-node
+            name: peer-node
+            friendlyName: 0f2ccdc
+            roles: 'Peer'
+            ssl:
+                privateKey: ●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●
+                publicKey: 0f2ccdc6d2e6e8012271ccb7f391ee79ef4b92fedc831936158076120edcddcc
+            signing:
+                # These are the keys and address of your node
+                privateKey: ●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●
+                publicKey: 5276BBE852DDBCBDB2343C4349083D055F0E6F19552E5E955B4207E90E45CD6F
+                address: TC7DOAQY65IPHI5NR7R4LYHYD3OEUD6PVDJISVA
+            vrf:
+                privateKey: ●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●
+                publicKey: 1BB864D80F9FC8BF661265276E00E57C1ABDD13B1454A3F5ADC025279FA03268
+                address: TD74YTRSVU4HXEXE4LQYIX2EQ65XLYO4V5XQB4I
+
+.. note:: Keep you Secret Keys secret at all times!
+
+Use the information in the ``signing`` section to access the node's account.
+
+***************************
+Providing funds to the node
+***************************
+
+Before the node can begin harvesting or voting (see below) it needs a **minimum amount of funds**. In the ``testnet`` test environment you can use |codename|'s **Faucet** to provide these funds.
```

Comment:
minimum amount (probably listed in the harvesting section in the doc):
- harvesting: 10k
- voting: 3 million  
Should it be mentioned here ??

Reply:
@Wayonb should we state the actual testnet number for harvesting and voting in the guide?

Reply:
I'd rather not have all this data spread over all the docs, so we don't need to update every page when the data changes.
Plus, this quantity is configurable, right?
So, yeah, I prefer mentioning it only in one place (the harvesting guide, probably).

Reply:
Fair enough... I also think those number could be tuned